core/sql/exp/exp_attrs.cpp

```diff
@@ -278,6 +278,8 @@ switch (datatype)
   case REC_INT_FRACTION: return extFormat? (char *)"INTERVAL FRACTION":(char *)"REC_INT_FRACTION";
   case REC_BLOB: return extFormat? (char *)"BLOB":(char *)"REC_BLOB";
   case REC_CLOB: return extFormat? (char *)"CLOB":(char *)"REC_CLOB";
+  case REC_BOOLEAN: return extFormat ? (char *)"BOOLEAN" : (char *)"BOOLEAN";
```

Comment:
The second string should be "REC_BOOLEAN" instead of "BOOLEAN". Will fix
that in a subsequent checkin.